src/osm_configurator/control/application_controller.py

```diff
@@ -10,12 +41,50 @@ class ApplicationController:
     def __init__(self):
         """
         Creates a new Application. It creates the view, the model and the control. It is responsible for starting
-        everything up and to switch to the normal workflow of the application.
+        everything up and to prepare the normal workflow of the application.
         """
-        pass
+        # Create Model
+        self._application: Application = application.Application()
 
-    def main(self) -> None:
+        # Create Control
+        self._aggregation_controller: AggregationController = aggregation_controller.AggregationController(
+            self._application)
+        self._calculation_controller: CalculationController = calculation_controller.CalculationController(
+            self._application)
+        self._category_controller: CategoryController = category_controller.CategoryController(self._application)
+        self._cut_out_controller: CutOutController = cut_out_controller.CutOutController(self._application)
+        self._data_visualization_controller: DataVisualizationController = data_visualization_controller\
+            .DataVisualizationController(self._application)
+        self._export_controller: ExportController = export_controller.ExportController(self._application)
+        self._osm_data_controller: OSMDataController = osm_data_controller.OSMDataController(self._application)
+        self._project_controller: ProjectController = project_controller.ProjectController(self._application)
+        self._settings_controller: SettingsController = settings_controller.SettingsController(self._application)
+
+        # Create View
+        self._main_window: MainWindow = main_window.MainWindow(
+            self._export_controller,
+            self._category_controller,
+            self._project_controller,
+            self._settings_controller,
+            self._aggregation_controller,
+            None,
+            self._calculation_controller,
+            self._cut_out_controller,
+            self._data_visualization_controller,
+            self._osm_data_controller)
+
+    def start(self):
         """
-        Starts the application.
-        This class method's only job is, to give control to an instance of the ApplicationController.
+        Starts up the Application. Enters the normal workflow of the application
         """
+        pass
+        # self._main_window.start()
```

Comment:
sth is not right here. if it should be like this add a devloper comment why it should be like this.